action.yml

```diff
@@ -1,6 +1,10 @@
 name: 'Run Julia package tests'
 description: 'Run the tests in a Julia package'
 author: 'David Anthoff'
+inputs:
+  inline:
+    description: 'Value passed to the --inline flag. Options: yes | no. Default value: yes.'
+    default: 'yes'
```

Comment:
I'm wondering whether we should make this `true` and `false` here? Are we generally using `true` and `false` for flags in these actions?

Reply:
I think using the same interface that the CLI uses makes sense here, even if the default for actions is different (I don't know if it is).